docs/JLBP-0006.md

```diff
@@ -11,7 +11,7 @@ repository system, library B needs two identifiers to find classes in library A:
 1. The Maven coordinates of library A, following the form
    `group_ID:artifact_ID:version`; for example
    `com.google.guava:guava:26.0-jre`. The Maven coordinates are used to locate the
-   library's jar in a Maven repository. For each pair of group ID and artifact ID
+   library's JAR file in a Maven repository. For each pair of group ID and artifact ID
```

Comment:
"library's JAR file" --> library's files 

(this is used for POMs, signatures, and other non-JAR files too)

Reply:
Yes, updated.